addons/mail/static/src/js/tours/discuss_channel_tour.js

```diff
@@ -72,19 +72,5 @@ registry.category("web_tour.tours").add("discuss_channel_tour", {
             tooltipPosition: "bottom",
             run: "click",
         },
-        {
-            trigger: ".o-mail-DiscussSidebarCategory-chat .o-mail-DiscussSidebarCategory-add",
-            content: markup(
-                _t(
-                    "<p><b>Chat with coworkers</b> in real-time using direct messages.</p><p><i>You might need to invite users from the Settings app first.</i></p>"
-                )
-            ),
-            tooltipPosition: "bottom",
-            run: "click",
-        },
-        {
-            isActive: ["auto"],
-            trigger: ".o-discuss-ChannelSelector",
-        },
```

Comment:
Instead of removing the steps, it should re-open the search for user/chat instead of channel.